### PR TITLE
added _isDisabledWhenAccessibilityActive

### PR DESCRIPTION
**Pull request by moloko**
plus a general tidy-up

### CODE REVIEW

File: js/adapt-articleView.js [hunk 1]
```diff
@@ -27,10 +27,14 @@ define([
         },
 
         preRender: function() {
-
             AdaptArticleView.prototype.preRender.call(this);
-            if (this.model.isBlockSliderEnabled()) this._blockSliderPreRender();
 
+            if (this.model.isBlockSliderEnabled()) {
+                this._blockSliderPreRender();
+                this.$el.addClass('abs-enabled');
```
Comment (tomgreenfield) [inline js/adapt-articleView.js]: You could define classes in a `className` function:
```js
className: function() {
    return AdaptArticleView.prototype.className.call(this) +=
        this.model.isBlockSliderEnabled() ? ' abs-enabled' : ' abs-disabled';
}
```
Also, is this at odds with line 86?

File: js/adapt-articleModel.js [hunk 2]
```diff
@@ -1,14 +1,21 @@
...
 
-	var BlockSliderModel = {
+    var BlockSliderModel = {
 
-		isBlockSliderEnabled: function() {
-			return this.get("_articleBlockSlider") && this.get("_articleBlockSlider")._isEnabled;
-		}
+        isBlockSliderEnabled: function() {
+            var config = this.get('_articleBlockSlider');
+            if (config && config._isEnabled) {
```
Comment (tomgreenfield) [inline js/adapt-articleModel.js]: Return early to sink the indentation?
```js
if (!config || !config._isEnabled) return;
```

Comment (oliverfoster) [inline js/adapt-articleModel.js]: return early will reduce the number of lines
```js
if (!config || !config._isEnabled) return false;
if (config._isDisabledWhenAccessibilityActive && Adapt.accessibility.isActive()) return false;
return true;
```

**Review by oliverfoster — CHANGES_REQUESTED** [review]

File: js/adapt-articleView.js (hunk 1 above)
Comment (moloko) [reply js/adapt-articleView.js]: I wouldn't say 'at odds' but there's certainly some duplication when ABS enabled

**Review by oliverfoster — APPROVED** [review]

**Review by tomgreenfield — APPROVED** [review]
Code eyeball.

**Pull request merged**